### ISSUE TITLE
Recently added constants aren't supported: ZMQ_RECONNECT_STOP_CONN_REFUSED ZMQ_RECONNECT_STOP_HANDSHAKE_FAILED ZMQ_RECONNECT_STOP_AFTER_DISCONNECT

### ISSUE THREAD

**Issue by yurivict** [zeromq/pyzmq#1789]
I wanted to try and see if they would solve the lack of error messages from ZMQ for TCP peers that can't be connected to but these constants are missing.

**Comment by minrk** [comment]
You're right, these draft constants are not yet exported. draft support in pyzmq is often incomplete.

In the meantime, you can use their actual values (1, 2, 3). pyzmq doesn't need to be aware of most libzmq constants for them to be used.

**Comment by yurivict** [comment]
I tried to use them like this: ```socket.setsockopt(2, 1)```, but it throws an exception ```Invalid argument```.

**Comment by minrk** [comment]
I've never used these options, but I believe those are _values_ for the [ZMQ_RECONNECT_STOP option](https://github.com/zeromq/libzmq/blob/c73447b9e7fefb2c16d067ee4a50cc7e53e1a172/doc/zmq_setsockopt.txt#L771-L799):

```python
socket.setsockopt(zmq.RECONNECT_STOP, 1) # or 2 or 3
```
